string project_id
string query_text
string fulfillment_text
float64 intent_detection_confidence

dialogflow_ros/Intent intent
dialogflow_ros/Parameter[] parameters
dialogflow_ros/Context[] output_contexts
